typegate/src/runtimes/python_wasi/python_vm.ts

```diff
@@ -46,6 +54,28 @@ function promisify<T>(fn: CallableFunction): Promise<T> {
   });
 }
 
+// TODO: update to new version
+const pythonWasiReactorUrl =
+  "https://github.com/metatypedev/python-wasi-reactor/releases/download/v0.1.0/python3.11.1-wasi-reactor.wasm.tar.gz";
+const cachePathReactor = join(config.tmp_dir, "python3.11.1-wasi-reactor.wasm");
+
+const libPythonUrl =
+  "https://github.com/assambar/webassembly-language-runtimes/releases/download/python%2F3.11.1%2B20230223-8a6223c/libpython-3.11.1.tar.gz";
```

Comment:
https://github.com/vmware-labs/webassembly-language-runtimes/releases/download/python%2F3.11.4%2B20230714-11be424/libpython-3.11.4-wasi-sdk-20.0.tar.gz is the new one